sdk/node/Libplanet.Node.Executable/Program.cs

```diff
@@ -1,48 +1,62 @@
 using Libplanet.Node.API.Services;
 using Libplanet.Node.Extensions;
 using Libplanet.Node.Options.Schema;
+using Microsoft.AspNetCore;
 using Microsoft.AspNetCore.Server.Kestrel.Core;
 
-SynchronizationContext.SetSynchronizationContext(new());
-var builder = WebApplication.CreateBuilder(args);
-
-builder.Logging.AddConsole();
-
-if (builder.Environment.IsDevelopment())
+var builder = WebHost.CreateDefaultBuilder(args);
+var assemblies = new string[]
+{
+    "Libplanet.Node.Swagger",
+    "Libplanet.Node.Explorer",
+};
+builder.UseSetting(WebHostDefaults.HostingStartupAssembliesKey, string.Join(';', assemblies));
+builder.ConfigureLogging(logging =>
+{
+    logging.AddConsole();
+});
+builder.ConfigureKestrel((context, options) =>
 {
-    builder.WebHost.ConfigureKestrel(options =>
+    if (context.HostingEnvironment.IsDevelopment())
     {
         // Setup a HTTP/2 endpoint without TLS.
         options.ListenLocalhost(5259, o => o.Protocols =
             HttpProtocols.Http1AndHttp2);
-    });
-}
-
-// Additional configuration is required to successfully run gRPC on macOS.
-// For instructions on how to configure Kestrel and gRPC clients on macOS,
-// visit https://go.microsoft.com/fwlink/?linkid=2099682
-
-// Add services to the container.
-builder.Services.AddGrpc();
-builder.Services.AddGrpcReflection();
-var libplanetBuilder = builder.Services.AddLibplanetNode(builder.Configuration);
-
-var app = builder.Build();
-var handlerMessage = """
-    Communication with gRPC endpoints must be made through a gRPC client. To learn how to
-    create a client, visit: https://go.microsoft.com/fwlink/?linkid=2086909
-    """;
-var schema = await OptionsSchemaBuilder.GetSchemaAsync(default);
-
-// Configure the HTTP request pipeline.
-app.MapGrpcService<BlockChainGrpcService>();
-app.MapGrpcService<SchemaGrpcService>();
-app.MapGet("/", () => handlerMessage);
-app.MapGet("/schema", () => schema);
-
-if (builder.Environment.IsDevelopment())
+    }
+});
+builder.ConfigureServices((context, services) =>
 {
-    app.MapGrpcReflectionService().AllowAnonymous();
-}
+    services.AddGrpc();
+    services.AddGrpcReflection();
+    services.AddLibplanetNode(context.Configuration);
+});
+builder.Configure((context, app) =>
+{
+    app.UseRouting();
+    app.UseEndpoints(endPoint =>
+    {
+        var handlerMessage = """
+            Communication with gRPC endpoints must be made through a gRPC client. To learn how to
+            create a client, visit: https://go.microsoft.com/fwlink/?linkid=2086909
+            """;
+        string? schema = null;
+        endPoint.MapGet(string.Empty, async context =>
+        {
+            await context.Response.WriteAsync(handlerMessage);
+        });
+        endPoint.MapGet("/schema", async context =>
```

Comment:
This endpoint is not fit for versioning practics. Check out [gRPC Versioning Guide](https://learn.microsoft.com/en-us/aspnet/core/grpc/versioning?view=aspnetcore-8.0)

Reply:
> This endpoint is not fit for versioning practics. Check out [gRPC Versioning Guide](https://learn.microsoft.com/en-us/aspnet/core/grpc/versioning?view=aspnetcore-8.0)

Implemented like the code below
```csharp
app.MapSchemaBuilder("/v1/schema");
app.MapGet("/schema", context => Task.Run(() => context.Response.Redirect("/v1/schema")));
```